src/client/root.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable */
+// This file is full of hacks
+// Hot reload won't work with Root as a pure function for example
+
+import React, { Component } from 'react'
+import { Router } from 'react-router'
+
+// Patch a load of rubbish React Router errors
```

Comment:
This patch silences a tonne of non-breaking errors from RR3. (Which are never going to be fixed. This will have to stay in until we can figure out RR4.